src/com/content-stats/stats-event.js

```diff
@@ -77,31 +80,61 @@ export default class ContentStatsEvent extends Component {
 
 			var Data = [];
 			if ( EventMode ) {
-				Data.push(<div><span class="-title">Signups:</span> <span class="-value">{stats.signups}</span></div>);
-				Data.push(<div><span class="-title">Unique Authors:</span> <span class="-value">{stats.authors}</span></div>);
+				Data.push(<div class="-gap"><span class="-title">Number of Users:</span></div>);
+				Data.push(<div><span>Signups:</span> <span class="-value">{stats.signups}</span></div>);
+				Data.push(<div><span>Unique Authors:</span> <span class="-value">{stats.authors}</span></div>);
 
 				if ( EventMode >= 5 ) {
-					Data.push(<div class="-gap"><span class="-title">Submissions:</span> <span class="-value">{stats.jam+stats.compo+stats.craft}</span></div>);
-					Data.push(<div class="-indent"><span class="-title">Jam:</span> <span class="-value">{stats.jam}</span></div>);
-					Data.push(<div class="-indent"><span class="-title">Compo:</span> <span class="-value">{stats.compo}</span></div>);
-					Data.push(<div class="-indent"><span class="-title">Craft:</span> <span class="-value">{stats.craft}</span></div>);
-					Data.push(<div><span class="-title">Unfinished:</span> <span class="-value">{stats.unfinished}</span></div>);
-					Data.push(<div><span class="-title">Unpublished:</span> <span class="-value">{stats.unpublished}</span></div>);
-					Data.push(<div><span class="-title">Warmups:</span> <span class="-value">{stats.warmup}</span></div>);
+					Data.push(<div class="-gap"><span class="-title">Number of Submissions:</span> <span class="-value -title">{stats.jam+stats.compo+stats.craft}</span></div>);
+					let entries = [
+						stats.compo,
+						stats.jam
```

Comment:
I'm a fan of trailing comma on last item but not sure what Mike feels about it.

Reason is that additions to arrays don't need to change as  many lines (i.e. add comma to previous last line)

Reply:
I do both. Typically trailing commas show up in places that there's an expectation (or actuality) that the contents will change.

Reply:
Not adding trailing spaces, we probably wont run more sub-events any time soon.
And eventually this may need to be generalised for non LD events so it will probably pull the list from the api at that point.

Reply:
Again just me and airbnb style guide, and I think it is a rule that makes sense. But not a strong opinion.

Reply:
Yeah it's not a bad practice (I could probably be sold on it). What's especially good about it is you don't end up accidentally changing a line just to add a comma when things are inserted, so the patches I see are cleaner. I just wish it was legal JSON to do this (it's not, most parsers will error if you do this). TBD. If you keep poking me about this I might convert. :laughing: 

But back to the point: This stats page is very "just there" at the moment. It's not well laid out or structured. My approach was more "lets get all the stats we care about on the page, then decide the best layout".